src/app/ui/components/containers/headers/header.tsx

```diff
@@ -9,56 +8,65 @@ import { RouteUrls } from '@shared/route-urls';
 
 import { useDrawers } from '@app/common/hooks/use-drawers';
 import { useViewportMinWidth } from '@app/common/hooks/use-media-query';
-import { LeatherLogo } from '@app/components/leather-logo';
 import { NetworkModeBadge } from '@app/components/network-mode-badge';
 import { LeatherButton } from '@app/ui/components/button';
+import { LeatherLogo } from '@app/ui/components/containers/leather-logo';
 import { ArrowLeftIcon } from '@app/ui/components/icons/arrow-left-icon';
 import { HamburgerIcon } from '@app/ui/components/icons/hamburger-icon';
 
-import { AppVersion } from './app-version';
+import { AppVersion } from '../../../../components/app-version';
 
+function isSettingsClickable(pathname: RouteUrls) {
+  return (
+    pathname !== RouteUrls.RequestDiagnostics &&
+    pathname !== RouteUrls.Onboarding &&
+    pathname !== RouteUrls.BackUpSecretKey &&
+    pathname !== RouteUrls.SetPassword &&
+    pathname !== RouteUrls.SignIn
+  );
+}
+
+function isLeatherLogoClickable(pathname: RouteUrls) {
+  return (
+    isSettingsClickable(pathname) && pathname !== RouteUrls.Home // should be able to click logo on home
```

Comment:
We have this code disabling clicking the logo on the `HOME` route but we should be able to do that I guess?